lib/state_machines/integrations/active_record.rb

```diff
@@ -565,7 +565,13 @@ def locale_path
 
         # Defines a new named scope with the given name
         def create_scope(name, scope)
-          lambda { |model, values| model.where(scope.call(values)) }
+          ->(model, values) { values.present? ? model.where(scope.call(values)) : model.all }
+        end
+
+        # Generates the results for the given scope based on one or more states to filter by
+        def run_scope(scope, machine, klass, states)
```

Comment:
Why was this method added?

Reply:
sorry, I add it to override this method from [state machine](https://github.com/gagalago/state_machines/blob/79cc6dbb195d73cf730598621b4a4a1d9378c435/lib/state_machines/machine.rb#L2130-L2135) (see https://github.com/state-machines/state_machines/pull/63). 
I don't know how to change gem dependency to do that in a good way. Can you give me some information, so that I can to it?

Reply:
@diegotoral  Are you up to rebase this PR ?

Reply:
I am not updated with the pull request but I can give it a try some day this week. :+1:

Reply:
Any update ?

Reply:
I have the branch rebased but I'm not allowed to push it here. :grimacing:

Reply:
Can you open a new pre ?

Reply:
thanks to continuing my work :+1: